### ISSUE TITLE
Cells have no padding on `qutebrowser`

### ISSUE THREAD

**Issue by nikolai-cc** [theministry/feature-tracker#15]
On initial test column overflows don't seem to render correctly either

Version info:
```
         ______     ,,
    ,.-"`      | ,-` |
  .^           ||    |
 /    ,-*^|    ||    |
;    /    |    ||    ;-*```^*.
;   ;     |    |;,-*`         \
|   |     |  ,-*`    ,-"""\    \
|    \   ,-"`    ,-^`|     \    |
 \    `^^    ,-;|    |     ;    |
  *;     ,-*`  ||    |     /   ;;
    `^^`` |    ||    |   ,^    /
          |    ||    `^^`    ,^
          |  _,"|        _,-"
          -*`   ****"""``

qutebrowser v2.4.0
Git commit: 
Backend: QtWebEngine 5.15.8, based on Chromium 87.0.4280.144
Qt: 5.15.3 (compiled 5.15.2)

CPython: 3.10.2
PyQt: 5.15.6

sip: 6.5.1
colorama: 0.4.4
jinja2: 3.0.3
pygments: 2.11.2
yaml: 6.0
adblock: 0.5.2
PyQt5.QtWebEngineWidgets: yes
PyQt5.QtWebEngine: 5.15.5
PyQt5.QtWebKitWidgets: yes
pdf.js: no
sqlite: 3.38.0
QtNetwork SSL: OpenSSL 1.1.1m  14 Dec 2021

Style: QFusionStyle
Platform plugin: xcb
OpenGL: Intel, 4.6 (Compatibility Profile) Mesa 21.2.5
Platform: Linux-5.16.12-arch1-1-x86_64-with-glibc2.35, 64bit
Linux distribution: Arch Linux (arch)
Frozen: False
Imported from /usr/lib/python3.10/site-packages/qutebrowser
Using Python from /usr/bin/python3
Qt library executable path: /usr/lib/qt/libexec, data path: /usr/share/qt

Paths:
cache: /home/sliebau/.cache/qutebrowser
config: /home/sliebau/.config/qutebrowser
data: /home/sliebau/.local/share/qutebrowser
runtime: /run/user/1000/qutebrowser
system data: /usr/share/qutebrowser

Autoconfig loaded: yes
Config.py: no config.py was loaded
Uptime: 16:37:28

```

**Comment by nikolai-cc** [comment]
![image](https://user-images.githubusercontent.com/17435776/161270127-c146bfef-6100-4567-9381-8b9cd23cfe31.png)

**Comment by nikolai-cc** [comment]
should be fixed in e78462ef5e36bca27fb57d2e75d2bd670334d415
